liblangutil/Scanner.cpp

```diff
@@ -343,13 +366,13 @@ Token Scanner::scanMultiLineDocComment()
 	bool endFound = false;
 	bool charsAdded = false;
 
-	while (isWhiteSpace(m_char) && !isLineTerminator(m_char))
+	while (isWhiteSpace(m_char) && !isLineTerminator())
```

Comment:
Shall we do the same with `isWhiteSpace()` then?

Reply:
Probably in another PR ;)

Reply:
Right @erak. I'd prefer to stick with what this PR was about. OTOH, I also prefer free-functions, except that for isLineTerminator it needed more than just a byte, due to multi-byte line termination sequences. :)